.travis.yml

```diff
@@ -18,14 +18,9 @@ cache:
 git:
   depth: 5
 
-addons:
```

Comment:
(note to self, we don't need this because we tell intern what to use in its config)